styles.css

```diff
@@ -1,7 +1,91 @@
 @import './components/Tabs/Tabs.css';
 
-body {
+* {
 	box-sizing: border-box;
+}
+
+body {
 	min-width: 300px;
 	margin: 0;
 }
+
+section {
+	border: 1px solid black;
+}
+
+.dropdown {
+	position: absolute;
+	left: 10px;
+	top: 10px;
+}
+
+.dropdown__button {
+	background-color: white;
+	padding: 10px 0;
+	width: 200px;
+	font-size: 16px;
+	border: 1px solid black;
+}
+
+.dropdown__menu {
+	display: none;
+	margin-top: 3px;
+	background-color: white;
+	flex-direction: column;
+	width: 198px;
+	border: 1px solid black;
+}
+
+.dropdown__menu--show {
+	display: flex;
+}
+
+.dropdown__menu a {
+	text-decoration: none;
+	color: black;
+	padding: 10px 0;
+	text-align: center;
+}
+
+.dropdown__menu a:hover {
```

Comment:
I think I know what your favorite colors are - pink and teal (maybe purple?)! Haha, let me know if I'm wrong. I feel like those are the colors that you reach for the most. And I don't blame ya, they're pretty!